contracts/DecentralizedKV.sol

```diff
@@ -20,7 +23,7 @@ contract DecentralizedKV {
         /* Block Size. aligned with 2^n */
         uint24 kvSize;
         /* Commitment */
-        bytes24 hash;
+        bytes32 hash;
```

Comment:
I think we should still use bytes24 as a truncated root hash could still work for inclusion proof.